radio/src/opentx.h

```diff
@@ -580,6 +580,7 @@ PACK(struct GlobalData {
   uint8_t unexpectedShutdown;
   uint8_t usbConnect;
   uint8_t cToolRunning;
+uint8_t flyskygimbals;
```

Comment:
After suggested changes this can go back to flysky gimball driver space and keep the same name as in source (EdgeTX)

Reply:
Fixed!